lunatrace/bsl/ingest-worker/docker/queuehandler.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.18-alpine AS builder
+
+COPY --from=repo-bootstrap /usr/repo/ /build/
+WORKDIR /build/lunatrace/cli/
+
+# generate the graphql code from the schema
+RUN go generate ./...
```

Comment:
This should not be part of the build process. Workflow for go generate is to commit the generated files then build off that.